public/index.html

```diff
@@ -11,6 +11,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" />
     <title>React App</title>
+    <script type="module" src="https://unpkg.com/oboe@2.1.5/dist/oboe-browser.js"></script>
```

Comment:
Include the oboe in the HTML file to make it be global function.

Reply:
Can we add this as a dependency and have it bundled with the app?